packages/cosmos_ui_components/lib/components/cosmos_bottom_sheet_header.dart

```diff
@@ -42,7 +42,7 @@ class CosmosBottomSheetHeader extends StatelessWidget {
               ],
             ),
           ),
-        SizedBox(height: theme.spacingM),
+        SizedBox(height: isExtended ? theme.spacingXL : theme.spacingXXXL),
         Padding(
```

Comment:
since we already have padding widget below, we could simply add `top: isExtended ? theme.spacingXL : theme.spacingXXXL` and remove the sizedBox alltogether

Reply:
Updated